test/src/snapshot/mod.rs

```diff
@@ -77,40 +77,60 @@ pub(super) async fn run(filter_regex: Option<&regex::Regex>) -> Result<()> {
 
-                    // known commands
-                    let cmd = if let Some(cmd) = cmd.strip_prefix("forc doc ") {
-                        FORC_DOC_COMPILATION.call_once(|| {
-                            compile_forc_doc();
-                        });
-                        format!("target/release/forc-doc {cmd} 1>&2")
-                    } else if let Some(cmd) = cmd.strip_prefix("forc ") {
-                        FORC_COMPILATION.call_once(|| {
-                            compile_forc();
-                        });
-                        format!("target/release/forc {cmd} 1>&2")
-                    } else {
-                        panic!("Not supported. Possible commands: forc")
-                    };
-
-                    let o = duct::cmd!("bash", "-c", cmd.clone())
-                        .dir(repo_root.clone())
-                        .stderr_to_stdout()
-                        .stdout_capture()
-                        .env("COLUMNS", "10")
-                        .unchecked()
-                        .start()
-                        .unwrap();
-                    let o = o.wait().unwrap();
-
-                    let _ = writeln!(
-                        &mut snapshot,
-                        "{}",
-                        clean_output(&format!(
+                    let mut last_output: Option<String> = None;
+
+                    for cmd in cmd.split("|") {
+                        let cmd = cmd.trim();
+
+                        // known commands
+                        let cmd = if let Some(cmd) = cmd.strip_prefix("forc doc ") {
+                            FORC_DOC_COMPILATION.call_once(|| {
+                                compile_forc_doc();
+                            });
+                            format!("target/release/forc-doc {cmd} 1>&2")
+                        } else if let Some(cmd) = cmd.strip_prefix("forc ") {
+                            FORC_COMPILATION.call_once(|| {
+                                compile_forc();
+                            });
+                            format!("target/release/forc {cmd} 1>&2")
+                        } else if let Some(cmd) = cmd.strip_prefix("grep ") {
+                            let arg = cmd.trim();
+                            if let Some(l) = last_output.take() {
+                                let mut new_output = String::new();
+                                for line in l.lines() {
+                                    if line.contains(arg) {
+                                        new_output.push_str(line);
+                                        new_output.push('\n');
+                                    }
+                                }
+                                last_output = Some(new_output);
+                            }
+                            continue;
+                        } else {
+                            panic!("Not supported. Possible commands: forc")
```

Comment:
grep seems to be supported to no?

Reply:
`grep` is indeed supported. Well.... a very limited version of it, to simplify the snapshots.